.github/workflows/contributors.yml

```diff
@@ -0,0 +1,29 @@
+name: Maintaining Contributors List
+on:
+  pull_request:
+    branches: [main]
+
+jobs:
+  update-contributors:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Git checkout
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.head_ref }}
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: '*'
+          cache: 'npm'
+          cache-dependency-path: 'npm-shrinkwrap.json'
+          check-latest: true
+      - name: Install dependencies
+        run: npm ci --no-audit
+      - name: Generate Contributors
```

Comment:
We might be able to use https://github.com/actions/github-script#run-a-separate-file-with-an-async-function for this.

It will save us from installing the `octokit` dependencies directly.

Reply:
We'll need to wait for https://github.com/actions/github-script/issues/168 or use `commonJS` syntax